pyproject.toml

```diff
@@ -41,6 +41,7 @@ cairocffi = "^1.1.0"
 grpcio =  { version = "*", optional = true }
 grpcio-tools = { version = "*", optional = true }
 watchdog = { version = "*", optional = true }
+networkx = "^2.5"
```

Comment:
If adding a dependency my suggestion would be to make it optional.

For example, see how js-renderer's dependencies are done.

Reply:
I think the graph module should be a core feature of Manim, not an optional one. And as I've written above: if we are going to include a `Graph` class, I strongly suggest that we don't reinvent the wheel while doing so. There are a lot of graph-specific algorithms that would be cool to have access to; it does not stop at automatic layouting.